JuFlajollet/Gerez-un-projet-collaboratif-en-int-grant-une-demarche-CI-CD
# Feature - Create GitHubActions for Back and Front

Creating GitHub Actions for Back and Front applications.
These github actions do the following:
- checkout source code
- build application
- test application
- verify that application pass Sonar's quality gate
- package application
- create docker image
- push docker image to Docker Hub
